contracts/0.8.9/oracle/ValidatorsExitBus.sol

```diff
@@ -308,7 +327,6 @@ abstract contract ValidatorsExitBus is AccessControlEnumerable, PausableUntil, V
         _checkExitSubmitted(requestStatus);
         _checkDelivered(requestStatus);
         _checkExitRequestData(exitsData.data, exitsData.dataFormat);
-        _checkContractVersion(requestStatus.contractVersion);
```

Comment:
:thinking: @F4ever

Reply:
There was a check in the triggerExits function that could prevent triggering exits for validators requested to exit in older contract versions. This check was added by mistake and later removed.

Contract version should only be checked during data revealing.